src/wizard/android/index.md

```diff
@@ -5,44 +5,22 @@ support_level: production
 type: framework
 ---
 
-## Integrating the SDK
+## Install
 
-Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your app works.
-
-### Auto-Installation and Source Context With the Sentry Android Gradle Plugin
-
-The Sentry Android Gradle plugin will install the Android SDK and integrations relevant to your application.
-It can also upload your source code to Sentry so it can be shown as part of the stack traces.
-
-To install the plugin, please update your app's `build.gradle` file as follows:
+Add the [Sentry Android Gradle plugin](/platforms/android/gradle/) to your `app` module:
 
 ```groovy {filename:app/build.gradle}
-buildscript {
-  repositories {
-    mavenCentral()
-  }
-}
 plugins {
+  id "com.android.application" // should be in the same module
   id "io.sentry.android.gradle" version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.9.0') }}"
 }
-
-sentry {
-  // Generates a JVM (Java, Kotlin, etc.) source bundle and uploads your source code to Sentry.
-  // This enables source context, allowing you to see your source
-  // code as part of your stack traces in Sentry.
-  includeSourceContext = true
-
-  org = "___ORG_SLUG___"
-  projectName = "___PROJECT_SLUG___"
-  authToken = "your-sentry-auth-token"
-}
 ```
 
-The plugin version `{{@inject packages.version('sentry.java.android.gradle-plugin', '3.9.0') }}` will automatically add the Sentry Android SDK (version `{{@inject packages.version('sentry.java.android', '6.21.0') }}`) to your app.
+The plugin will automatically add the Sentry Android SDK (version `{{@inject packages.version('sentry.java.android', '6.21.0') }}`) to your app.
```

Comment:
I think we can remove the sdk version details, as it's usually not correct and adds little benefit. I remember being the one adding this, and I think I was wrong here.
> (version `{{@inject packages.version('sentry.java.android', '6.21.0') }}`)